File: Second Year (2013-2014)/OMGL7/TP2 - Website in PHP/musiconline/musiconline/db/Commande.php
<?php
/* Authors: ANDRIAMILANTO Tompoariniaina && CANNO Camille
* Groupe: B1
* File: Commande.php
*/

class Commande implements DataObject{
   private $NumeroCommande;
   private $ClientCommande;
   private $TitreCommande;
   private $Total;

   public function  __construct() { }

   public function init($NumeroCommande,$ClientCommande,$TitreCommande,$Total){
      $this->NumeroCommande=$NumeroCommande;
      $this->ClientCommande=$ClientCommande;
      $this->TitreCommande=$TitreCommande;
      $this->Total=$Total;
   }


   public function getNUmeroCommande(){
     return $this->NumeroCommande;
   }

   public function setNUmeroCommande($NumeroCommande){
     $this->NumeroCommande = $NumeroCommande;
   }


   public function getCLientCommande(){
     return $this->ClientCommande;
   }

   public function setCLientCommande($ClientCommande){
     $this->ClientCommande = $ClientCommande;
   }


   public function getTItreCommande(){
     return $this->TitreCommande;
   }

   public function setTItreCommande($TitreCommande){
     $this->TitreCommande = $TitreCommande;
   }


   public function getTOtal(){
     return $this->Total;
   }

   public function setTOtal($Total){
     $this->Total = $Total;
   }


}
?>

File: Second Year (2013-2014)/OMGL7/TP3 - Website in JSP/src/db/Commande.java
package db;
/**
* @author Andriamilanto Tompoariniaina - Canno Camille - Info2B
*/

/**
* Classe pour les objets Commande
*/
public class Commande {
	private int numeroCommande;
	private String clientCommande;
	private String titreCommande;
	private int total;
	 
	/**
	* Constructeur de la classe Commande
	*/
	public Commande() {}
	  
	/**
	* Méthode qui initialise les attributs de l'objet Commande
	*/
	public void init(int numCo, String cliCo, String titCo, int tot) {
		this.numeroCommande = numCo;
		this.clientCommande = cliCo;
		this.titreCommande = titCo;
		this.total = total;
	}
	  
	/**
	* Accesseur de l'objet
	* @return numeroCommande
	*/
	public int getNumeroCommande() {
		return this.numeroCommande;
	}
	   
	/**
	* Modificateur de l'objet
	*/
	public void setNumeroCommande(int numCo) {
		this.numeroCommande = numCo;
	}
	   
	/**
	* Accesseur de l'objet
	* @return clientCommande
	*/
	public String getClientCommande() {
		return this.clientCommande;
	}
		
	/**
	* Modificateur de l'objet
	*/
	public void setClientCommande(String cliCo){
		this.clientCommande = cliCo;
	}
		
	/**
	* Accesseur de l'objet
	* @return titreCommande
	*/
	public String getTitreCommande(){
		return this.titreCommande;
	}
	   
	/**
	* Modificateur de l'objet
	*/
	public void setTitreCommande(String titCo){
		this.titreCommande = titCo;
	}
	   
	/**
	* Accesseur de l'objet
	* @return total
	*/
	public int getTotal(){
		return this.total;
	}

	/**
	* Modificateur de l'objet
	*/
	public void setTotal(int tot){
		this.total = tot;
	}
}